postproengine/openfast.py

```diff
@@ -372,6 +379,19 @@ def running_average(data):
 
                 csvfile = os.path.join(output_dir, prefix + "_running_avg" + extension)
                 running_avg_df.to_csv(csvfile, index=False,float_format='%.15f')
+
+            if 'pitch_travel':
+                pitch_travel_columns  = filtered_df.filter(regex='Pitch',axis=1).columns
+                if pitch_travel_columns.empty:
+                    print("No variables found containing 'Pitch'. Add to array of vars.")
+                else:
+                    pitch_travel_df = pd.DataFrame(0,index=range(1),columns=pitch_travel_columns)
+                    for column in pitch_travel_columns:
+                        pitch_travel = np.sum(np.abs(np.asarray(filtered_df[column][mask].values)))
```

Comment:
Just to make sure I understand the calculation of blade pitch travel, should we calculate it as 

$$
\theta_{travel} = \sum_i | \theta_i |
$$

or

$$
\theta_{travel} = \sum_i | \theta_i - \theta_{i-1} | ?
$$

i.e., should there be a `np.diff` in this statement?